docs/devices/HG06106B.md

```diff
@@ -17,6 +17,15 @@ description: "Integrate your Lidl HG06106B via Zigbee2MQTT with whatever smart h
 
 ## Notes
 
+### Pairing
+Factory reset the light bulb.
+After resetting the bulb will automatically connect.
+
+While pairing, keep the bulb close to the CC2531 USB sniffer.
```

Comment:
Change CC2531 -> adapter since not everyone uses a CC2531. (also for the other PRS)

Reply:
it was just a copy of the ikea bulb pairing, I'll remove it